src/main/webapp/app/entities/result/updating-result.component.ts

```diff
@@ -73,6 +73,16 @@ export class UpdatingResultComponent implements OnChanges, OnDestroy {
             .pipe(
                 // Ignore initial null result of subscription
                 filter(result => !!result),
+                // TODO: remove logging
```

Comment:
- [ ] remove logging

Reply:
the logging is here to investigate why the submit button in the code editor does not work for the second time. I guess something is missing when we send the result to the client. Did you notice something in the console logs?

Reply:
No, nothing in the browser logs 😔